### ISSUE TITLE
Rkyv 0.6.3 depends on unstable feature?

### ISSUE THREAD

**Issue by syrusakbary** [djkoloski/rkyv#123]
Latest version of RKYV depends on an unstable feature `bool_to_option`

This is causing Wasmer benchmarks to fail:
https://github.com/wasmerio/wasm-runtime-benchmarks/runs/2511212981?check_suite_focus=true

```
error[E0658]: use of unstable library feature 'bool_to_option'
  --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/rkyv_derive-0.6.3/src/archive.rs:49:28
   |
49 |     let strict = is_strict.then::<Attribute, _>(|| parse_quote! { #[repr(C)] });
   |                            ^^^^
   |
   = note: see issue #64260 <https://github.com/rust-lang/rust/issues/64260> for more information
```

It would be great if we can use an API that will work in stable versions of Rust.

**Comment by syrusakbary** [comment]
It seems this feature was stabilized in Rust 1.50. So we just need to update it!
I'll keep this issue open so we can update it once the README reflects this min Rust version

**Comment by djkoloski** [comment]
Fixed by 44e0e05c698479d986781dd556f7fa6fb3f1718f
